src/test/scripts/helm_uninstall.sh

```diff
@@ -15,12 +15,20 @@ helm uninstall -n "${TARGET_NAMESPACE}" "${FUNCTEST_RELEASE_NAME}"
 helm uninstall -n "${TARGET_NAMESPACE}" "${PRODUCT_RELEASE_NAME}"
 helm uninstall -n "${TARGET_NAMESPACE}" "${POSTGRES_RELEASE_NAME}"
 
-# delete any and all persistent volume claims by label
+if [ "$START_NFS_SERVER" != true ]; then
+  # we are using a "shared shared home", remove the subdirectory of the shared-home
+  "$THISDIR"/shared_home_browser_install.sh "${TARGET_NAMESPACE}"
+  kubectl exec -n "${TARGET_NAMESPACE}" shared-home-browser -- rm -rf "/shared-home/$PRODUCT_RELEASE_NAME"
+fi
+
+# delete any and all persistent volumes and claims by label
+# todo DCNG-947
+echo Deleting PVCs, if the uninstall script gets stuck deleting the shared pvc here, run:
+sharedPvcName=$(kubectl get -n dcng pvc -l app.kubernetes.io/instance=pbruski-bitbucket | grep shared | awk '{print $1;}')
+echo kubectl patch pvc -n "${TARGET_NAMESPACE}" "$sharedPvcName" -p "'{\"metadata\":{\"finalizers\":null}}'"
```

Comment:
Why is the below line not enough to delete the PVC?

Reply:
What I saw several times was that for no good reason, the pods/PVs/PVCs were stuck in terminating state. This was probably caused by incorrect order of deletion. When that happens, the finalizers need to be removed from the object. I want to see if that happens in CI too, if it does then we'll need to come up with a solution.

Reply:
BTW, this is not specific to NFS, earlier I saw that with EFS too.

Reply:
Deleting a PV first should solve the problem, though making a PVC unprotected via the above patch work too. Yes, it's not related to storage backend.